harness/determined/cli/dev.py

```diff
@@ -0,0 +1,77 @@
+import json
+import shlex
+import shutil
+import subprocess
+import sys
+from argparse import Namespace
+from typing import List
+
+from termcolor import colored
+
+from determined.common.api import authentication
+from determined.common.api.request import make_url
+from determined.common.declarative_argparse import Arg, Cmd
+
+
+@authentication.required
+def token(_: Namespace) -> None:
+    assert authentication.cli_auth is not None
```

Comment:
Does this give a nice message or a stack trace?

Reply:
how about we just do
`token = authentication.must_cli_auth().get_session_token()`